### PR TITLE
Document client timeout in query management properties

### PR DESCRIPTION
**Pull request by colebow**
## Description

This is a simple fix to include the property for client timeout. Noted that it probably shouldn't be changed, but none of the other properties are editorialized, so I stuck with that and decided not to include a note to that extent.

> Is this change a fix, improvement, new feature, refactoring, or other?

Improvement

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Documentation

> How would you describe this change to a non-technical end user or system administrator?

We are adding documentation!

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by Colebow** [comment]
image

**Comment by Colebow** [comment]
image

File: docs/src/main/sphinx/admin/properties-query-management.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 Query management properties
 ===========================
 
+``query.client.timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Default value:** ``10m``
+
+Configures how long the cluster will run without contact from the client before
```
[the remark this answers is not in the document]
Comment (mosabua) [reply docs/src/main/sphinx/admin/properties-query-management.rst:11]: also maybe use "client application, such as the CLI, before"

**Review by mosabua — CHANGES_REQUESTED** [review]

**Review by mosabua — APPROVED** [review]

File: docs/src/main/sphinx/admin/properties-query-management.rst [hunk 2]
```diff
@@ -2,6 +2,15 @@
 Query management properties
 ===========================
 
+``query.client.timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Default value:** ``10m``
```
Comment (electrum) [inline docs/src/main/sphinx/admin/properties-query-management.rst:9]: Should be `5m`

**Review by electrum — APPROVED** [review]

**Comment by colebow** [comment]
@electrum updated the value, can we get this merged?

**Review by jhlodin — APPROVED** [review]

**Comment by mosabua** [comment]
@martint can we get this merged?

**Pull request merged**